src/core/Breadcrumbs/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { CommonProps } from '../utils/props';
+import { useTheme } from '../utils/hooks/useTheme';
+import { Button } from '../Buttons/Button';
+import '@itwin/itwinui-css/css/breadcrumbs.css';
+
+type BreadcrumbItemOwnProps<E extends 'a' | 'button' | 'span'> = {
+  /**
+   * Control which element the breadcrumb item should use.
+   *
+   * Use 'button' if you specify an `onClick` handler.
+   * Use 'a' if you also specify a meaningful `href`.
+   * Use 'span' if the item should not be clickable (e.g. for the current item).
+   * @default 'button'
+   */
+  element?: E;
+  /**
+   * Content of the breadcrumb item.
+   */
+  children: React.ReactNode;
+} & CommonProps;
+
+export type BreadcrumbItemProps<
+  E extends 'a' | 'button' | 'span'
+> = BreadcrumbItemOwnProps<E> &
+  Omit<React.ComponentPropsWithoutRef<E>, keyof BreadcrumbItemOwnProps<E>>;
+
+/**
+ * Individual breadcrumb component to be used in the `Breadcrumbs` items array.
+ * @example
+ * const items = [
+ *   <BreadcrumbItem>Root</BreadcrumbItem>,
+ *   <BreadcrumbItem>Item 1</BreadcrumbItem>,
+ *   <BreadcrumbItem>Item 2</BreadcrumbItem>,
+ * ];
+ * @example
+ * const items = [
+ *   <BreadcrumbItem element='a' href='/'>Root</BreadcrumbItem>,
+ *   <BreadcrumbItem element='a' href='/page1'>Page 1</BreadcrumbItem>,
+ *   <BreadcrumbItem element='a' href='/page1/sub1'>Sub page 1</BreadcrumbItem>,
+ * ];
+ */
+export const BreadcrumbItem = <E extends 'a' | 'button' | 'span' = 'button'>(
+  props: BreadcrumbItemProps<E>,
+) => {
+  const { className, children, element = 'button', ...rest } = props;
+
+  const Element = element === 'button' ? Button : element;
+
+  useTheme();
+
+  return (
+    <Element
```

Comment:
Other libs advertise a lot integration with react-router... Would it work here too?

Reply:
Added React.ComponentType so it will work now

Reply:
Does that mean that simple 'div' would not work?

Reply:
We only want users to be passing these 3 elements:
![image](https://user-images.githubusercontent.com/9084735/126996062-7f1df010-9443-4d4d-bbc9-2bb1787ea3a9.png)

For anything else, it should be custom component which needs to handle styles.

If we allow passing any arbitrary element, I'm afraid they will break things.

Reply:
Simple div will work now after the latest css changes

Reply:
Removed breadcrumb item